src/app/[lang]/send-mail-restore-pass/page.tsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import { getDictionary } from "../dictionaries";
+
+export default async function RestorePassword({
+  params,
+}: {
+  params: { lang: "uk" | "en" };
+}) {
+  const dict = await getDictionary(`/${params.lang}/send-mail-restore-pass`);
+
+  return (
+    <div className="container mx-auto px-6">
+      <h2 className="mt-20 text-center text-lg font-medium text-neutral-50">
+        {dict.checkEmail}
+      </h2>
+      <p className="mt-5 text-center font-main text-sm font-normal leading-tight text-neutral-50">
+        {dict.followLink}
+      </p>
+      <div className="mt-20 flex justify-center">
+        <Image src="/send-email.svg" alt="mail" width={226} height={180} />
```

Comment:
You can import local images and Next will automatically determine their sizes, preventing possible layout shifting that can occur when setting image sizes manually.

Make the same fix in the `src/app/[lang]/send-mail/page.tsx` file

```suggestion
import mail from "../../../../public/send-email.svg";

export default async function RestorePassword({
  params,
}: {
  params: { lang: "uk" | "en" };
}) {
  const dict = await getDictionary(`/${params.lang}/send-mail-restore-pass`);

  return (
    <div className="container mx-auto px-6">
      <h2 className="mt-20 text-center text-lg font-medium text-neutral-50">
        {dict.checkEmail}
      </h2>
      <p className="mt-5 text-center font-main text-sm font-normal leading-tight text-neutral-50">
        {dict.followLink}
      </p>
      <div className="mt-20 flex justify-center">
        <Image src={mail} alt="mail" />

```